whos-online/whos-online.js

```diff
@@ -0,0 +1,56 @@
+friends = [
+  {
+    username: "Mark",
+    status: "online",
+    lastActivity: 5
+  },
+  {
+    username: "Lucy",
+    status: "offline",
+    lastActivity: 22
+  },
+  {
+    username: "Bob",
+    status: "online",
+    lastActivity: 104
+  }
+];
+
+friends.forEach(friend => {
```

Comment:
This part should be inside `whosOnline`, because it's part of the solution and depends on `friends`, which is a parameter of that function.